calamari_ocr/ocr/datasets/dataset_factory.py

```diff
@@ -20,6 +20,7 @@ def create_dataset(type: DataSetType,
                    non_existing_as_empty=False,
                    args: dict = None,
                    ):
+    
```

Comment:
Remove blank line (PR should not change the layout of unaffected lines)

Reply:
done